src/resolvers/UserTag/usersAssignedTo.ts

```diff
@@ -0,0 +1,135 @@
+import { UserTagResolvers } from "../../types/generatedGraphQLTypes";
+import { TagUser, Interface_TagUser, Interface_User } from "../../models";
+import { validatePaginationArgs } from "../../libraries/validators/validatePaginationArgs";
+import { graphqlConnectionFactory } from "../../utilities/graphqlConnectionFactory";
+import { errors, requestContext } from "../../libraries";
+import { INVALID_CURSOR_PROVIDED } from "../../constants";
+
+// @ts-ignore
+export const usersAssignedTo: UserTagResolvers["usersAssignedTo"] = async (
+  parent,
+  args
+) => {
+  // Check that the provided arguments must either be correct forward pagination arguments or correct backward pagination arguments
+  validatePaginationArgs(args);
+
+  let allusersAssignedTo: Interface_TagUser[] | null;
+
+  const connectionObject = graphqlConnectionFactory<Interface_User>();
+
+  // Forward pagination
+  if (args.first) {
+    // Fetch the users
+    allusersAssignedTo = await TagUser.find({
+      ...(args.after && {
+        _id: {
+          $gte: args.after,
```

Comment:
technically both `_id` and `userId` can be used for filtering here, but `userId` would make more sense here, as that's what is intended for.

Reply:
Earlier the implementation worked on `userId` as the cursor itself, but in [the previous comments](https://github.com/PalisadoesFoundation/talawa-api/pull/1187#issuecomment-1481407955) you had mentioned that we should be sorting by the `createdAt` / `_id` property of the records. That's why I had shifted to using the record `_id` as the cursor.

Reply:
@EshaanAgg wasn't that about sorting by `userId`? Here I'm talking about filtering by `userId`. Filtering using `tagId` and `userId` would work because there combination is unique. Filtering using `tagId` and `_id` would work too, because `_id` by itself is unique, so using `tagId` serves no purpose here anymore.

Technically you only have the convinience of using `_id` because mongodb auto generates it, it's not necessary to have `_id` field for this because this document will only ever be used in context of `tagId` and `userId`.

Reply:
@xoldyckk 
But shouldn't we be filtering and sorting by the same property?
Otherwise won't be have a conflict as to which property should be the cursor?

Reply:
If we are to implement the sorting by `_id` and filtering by `userId`, then in the request, we would have access to only one of these properties as the cursor. 
And obviously it can't be ensured that if two documents in the `UserTag` schema (say d1 and d2) have d1.id >= d2.id, then the same imply d1.userId >= d2.userId.